contents/handbook/product/user-feedback.md

```diff
@@ -11,20 +11,10 @@ Gathering user feedback is critical for the development of our product. It's not
 > 😍 Want to share feedback? [File an issue](https://posthog.com/posthog/posthog/issues) or reach out on [Slack](/slack). **We're always happy to hear from you!**
 
 
-### Feedback heroes
-> **Historic context:** Previously a single person owned the feedback process. As we grew, we created the Feedback Hero ([see discussion](https://github.com/PostHog/product-internal/issues/98)) to scale our feedback process. On November 2021, we decided to remove the figure of "Feedback Hero" as multiple people now talk to users continuously for a variety of reasons.  
+### Beta Testers
+We have a group in our community slack called #beta-testers this is a group of engaged community users and paying customers who are keen to give feedback on our product in order to improve the product for themselves and other users.
```

Comment:
We also have a cohort of "Beta Users" (we may want to rename for consistency) which contain users who have explicitly expressed interest in being included in beta features. In the feature release process, it makes sense to release to them too.

Reply:
Yep - agreed, ideally we should try and seed the slack group with the folks from the Beta Users group. Would you be able to help with that @paolodamico ?